test/smoke/appServers/global-resources/Micrometer_ApplicationInsights.json

```diff
@@ -1,12 +1,8 @@
 {
-  "instrumentationSettings": {
-    "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://fakeingestion:60606/",
-    "preview": {
-      "instrumentation": {
-        "micrometer": {
-          "reportingIntervalMillis": 5000
-        }
-      }
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://fakeingestion:60606/",
+  "instrumentation": {
+    "micrometer": {
+      "reportingIntervalMillis": 5000
```

Comment:
should we be consistent with the interval unit?  for heartbeat, it's seconds, here it's milliseconds..  can we assume seconds for all intervals, and then simplify names to "interval"?

Reply:
ya, i like that, i will change this to seconds also. i think probably still keep suffix "Seconds" because it makes it more clear what users should put there

Reply:
done